contracts/interfaces/IMarketExtended.sol

```diff
@@ -41,8 +41,10 @@ interface IMarketExtendedCore is IMarketCommon {
   /// @notice Parameters when funding rate is updated
   /// @dev Can only be called by the current admin.
   struct FundingRateUpdate {
-    uint256 prevMultiplier;
-    uint256 newMultiplier;
+    uint128 prevMultiplier;
+    uint128 newMultiplier;
+    uint128 prevMinFloatPoolFundingBoost;
+    uint128 newMinFloatPoolFundingBoost;
```

Comment:
This struct is emitted in an event when the funding rate changes. We added the `minFloatPoolFundingBoost` value which should mostly stay constant, but can be changed by admin.

Struct isn't used anywhere else in code (literally just inside the event emitted by the `changeMarketFundingRateMultiplier` function).